config.json

```diff
@@ -23,6 +23,12 @@
       ]
     },
     {
+      "slug": "run-length-encoding",
+      "difficulty": 1,
+      "topics": [
```

Comment:
This depends. As it stands this is a very simple exercise, but if some of the more complicated error-handling and number-handling ideas (here https://github.com/exercism/x-common/issues/238) are implemented it would no longer be simple. I put it before `rna-transcription` because at the moment it only requires simple types and not `Maybe` for error handling.